packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+# Nimble Chip
+
+## Overview
+
+The `nimble-chip` is a component that presents text, an optional "remove" button, optional slotted content to the left of the text, and an optional error icon.
+
+### Background
+
+- [Nimble Chip component issue](https://github.com/ni/nimble/issues/413)
+- [Chip design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=9269-229459&m=dev)
+- [Spright chat feature backlog](https://github.com/ni/nimble/issues/2610) (component for attachment chips)
+
+### Containing Library
+
+The intent is for this to be put in Nimble, for the following reasons:
+
+- This is a fairly constrained component, and thus easily ported to any needed UI platform.
+- It has broad use possibilities.
+- Designs for the component have been completed (see above link)
+
+However, there is a potential for there to be some accessibility concerns, notably around the remove button, that could persuade us to put this in Spright.
+
+### Non-goals
+
+- Arbitrary HTML content
+- Implementing a "chip container" component that would be responsible for laying out the chips in a particular fashion, as well as possibly managing their removal if the "remove" button on a chip is pressed. While this may be useful at some point it is not necessary for the `Chip` component to have value.
+
+### Features
+
+- Optionally removable
+    - Note that the `Chip` won't remove itself, and it will be up to the client to remove it from the DOM.
+- Allows slotted content to be placed to left of text content. Slotted content will be styled automatically to be appropriately sized for the chip (i.e. given a height of `controlSlimHeight`). Types of slotted content that will be documented as acceptible are:
+    - icons
+    - buttons with `content-hidden` set and `appearance` set to `ghost`.
```

Comment:
What's the use-case for having buttons slotted in the `start` slot? Are there designs? I can't think of any other control (though may be forgetting one) that allows putting a button in the `start` slot.

Reply:
Yeah, I don't think there is a use-case for this. Ultimately, if anything, we will make the chip itself interactive (and potentially an actual button). Updated.